### ISSUE TITLE
The results of validating a model which training with tridentnet_r101v2c4_c5_multiscale_addminival_3x_fp16.py are bad.

### ISSUE THREAD

**Issue by zhayanli** [TuSimple/simpledet#327]
**Describe the bug**
Thanks for your excellent work！
I trained my own dataset using tridentnet. I converted my dataset to coco and ran create_coco_roidb.py , then changed gpus, num_class,log_frequency to 50,loader_worker to 4 in tridentnet_r101v2c4_c5_multiscale_addminival_3x_fp16.py. Do I need to change another parameters? I trained 10 epochs and used detection_test.py to validate. The results are bad.

Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.001
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.001
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.028
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.075
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.087
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.033
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.072
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.092

        I recorded RpnL1,RcnnL1 and Lr during the training. Could you help me, thank you!

![image](https://user-images.githubusercontent.com/40348284/82419464-0c089780-9ab1-11ea-9d29-e9710a948f43.png)
![image](https://user-images.githubusercontent.com/40348284/82419497-175bc300-9ab1-11ea-8f65-e03b3aace9ca.png)
![image](https://user-images.githubusercontent.com/40348284/82419537-2478b200-9ab1-11ea-9808-7df52370258c.png)

**Software info**
driver, CUDA, cuDNN versions
OS verison
My Software info is Ubantu 16.04,CUDA 10.0,cuDNN 7.4.2 

**How did you set up your MXNet for SimpleDet**

**Additional context**

**Comment by dongjuns** [comment]
How many images do you have?

**Comment by zhayanli** [comment]
50 thousands for training and 5 thousands for test

**Comment by dongjuns** [comment]
Thanks, and how many labels are in your training dataset?

**Comment by zhayanli** [comment]
90 thousands labels for 18 classes.

**Comment by RogerChern** [comment]
Could you please kindly share a part of your annotation and the prediction
json files?
